ui_framework/src/components/toast/_toast.scss

```diff
@@ -2,20 +2,27 @@
   @include kuiBottomShadow;
 
   position: relative;
-  padding: $kuiSizeM;
+  padding: $kuiSize;
   background-color: $kuiColorEmptyShade;
   border: $kuiBorderThin;
+  width: $kuiToastWidth;
```

Comment:
I thought of that too, but I decided to just put in both places for no other reason so the docs looked better (since they load them individually).